bedrock/products/templatetags/misc.py

```diff
@@ -300,33 +302,84 @@ def relay_bundle_countries():
     return settings.VPN_CONNECT_COUNTRIES
 
 
-# Relay has 4 plans
-# free
-# email
-# phone
-# bundle
+def _relay_make_ga_data(id, product, country_code, lang, plan):
+    plan = "monthly" if plan == "monthly" else "monthly_when_yearly"
+    currency = _relay_get_currency(product, country_code, lang)
+    if product == "relay-bundle":
+        # the bundle doesn't have a monthly price, discount calculated in reverse from % savings
+        yearly_when_yearly = relay_monthly_price("relay-bundle", RELAY_12_MONTH_PLAN, country_code, lang) * 12
+        discount = 55.92
+    elif product == "relay-phone":
+        monthly_when_monthly = relay_monthly_price("relay-phone", "monthly", country_code, lang)
+        monthly_when_yearly = relay_monthly_price("relay-phone", RELAY_12_MONTH_PLAN, country_code, lang)
+        yearly_when_yearly = monthly_when_yearly * 12
+        discount = 0 if plan == "monthly" else ((monthly_when_monthly - monthly_when_yearly) * 12)
+    elif product == "relay-email":
+        monthly_when_monthly = relay_monthly_price("relay-email", "monthly", country_code, lang)
+        monthly_when_yearly = relay_monthly_price("relay-email", RELAY_12_MONTH_PLAN, country_code, lang)
+        yearly_when_yearly = monthly_when_yearly * 12
+        discount = 0 if plan == "monthly" else ((monthly_when_monthly - monthly_when_yearly) * 12)
```

Comment:
Since these `relay-phone` and `relay-email` blocks are identical other than the string that's already defined in the `product` variable, these could collapse into one and pass the `product` variable instead of the string. Unless you envision future differences here. It could simplify this to a single `else` block.

Reply:
Thank you, that duplication was bothering me and for some reason I couldn't see the simple solution.